code/addons/test/src/node/reporter.ts

```diff
@@ -196,7 +196,7 @@ export class StorybookReporter implements Reporter {
         unhandledErrors[0]
       );
     } else {
-      const isCancelled = this.ctx.isCancelling;
+      const isCanceled = this.ctx.isCancelling;
```

Comment:
```suggestion
      const isCanceled = this.ctx.isCanceling;
```
This one too? Of course it needs to be updated where this value originates as well.

Reply:
I believe this is coming from the Vite code, not ours (based on a search of our monorepo)